src/auth/auth-get-domain.ts

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
This file should be renamed to reflect the change in the name of the class it exports. It also has a stray empty line at the top of the file that should be removed.